sonaranalyzer-dotnet/rspec/cs/S2183_c#.html

```diff
@@ -15,4 +15,14 @@
   return a &lt;&lt; 16;
 }
 </pre>
-
+<h2>Exceptions</h2>
```

Comment:
I don't think we made this change yet.
BTW, second bullet point is very good, but not sure if we want to make exception for the first bullet point - we do not use symbolic execution here, so the rule would be largely worthless.

Reply:
Shit it was supposed to be removed...